src/_error/getErrorFile.ts

```diff
@@ -15,14 +15,6 @@ export function getErrorFile({
   });
 
   if (customErrorFile) {
-    // Assign "origGetInitialProps" to custom _error page if needed
-    if (!customErrorFile.client.default.getInitialProps) {
-      const defaultErrorFile = getDefaultErrorFile({ nonIsolatedModules });
-      customErrorFile.client.default.getInitialProps =
-        defaultErrorFile.client.default.origGetInitialProps;
-      customErrorFile.server.default.getInitialProps =
-        defaultErrorFile.server.default.origGetInitialProps;
-    }
```

Comment:
When have we introduced this? It looks totally new to me 😄

Reply:
It was introduced in previous PR with initial 404 implementation. When custom error page has no `getInitialProps`, we should still provide it with the default props.

However, its not smart to patch "cached" modules like this (it leaks into other tests).